File: Euler/1/java/main.java
public class main {
  public static void main(String[] args) {
      int arg = Integer.parseInt(args[0]);
      int sum = 0;

      while (arg-- > 0) {
        if (arg % 3 == 0 || arg % 5 == 0) {
            sum += arg;
        }
      }

      System.out.println(sum);
  }
}

File: Euler/1/go/main.go
package main

import (
	"fmt"
	"os"
	"strconv"
)

func main() {
	var sum int64

	i, _ := strconv.ParseInt(os.Args[1], 0, 64)

	for i > 0 {
		i -= 1
		if i%3 == 0 || i%5 == 0 {
			sum += i
		}
	}

	fmt.Println(sum)
}
